### DISCUSSION TITLE
Prometheus and InfluxDB

### DISCUSSION THREAD

**Discussion by asdecopes** [OpenDTU-App/opendtu-react-native#93] [General]
Hi,
Is there a possibility to give support to InfluxDB as well as Prometheus DB? 
My Home Assistant is running under InfluxDB and I would prefer to stay with it and not change.

Danke!

**Comment by CommanderRedYT** [comment]
Hi,

yes, it is already planned to support multiple data sources. The problem is with ones that aren't directly supported by OpenDTU, which would be every DB except Prometheus.

With Prometheus, I can just implement the queries as they are described by the firmware itself. They will be the same for every user. However, with other DBs this isn't as easy, especially for the SQL ones.

Also, in your case it would probably make more sense to support homeassistant itself as a data source, however I am not sure what the requirements for this are.

Feel free to make suggestions, and if you feel like it, maybe als a PR!
